portfolio/src/main/java/com/google/sps/servlets/ListQuestionsServlet.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.repackaged.com.google.gson.Gson;
+
+@WebServlet("/list-questions")
+public class ListQuestionsServlet extends HttpServlet{
+    /**
+     *
```

Comment:
Did you mean to add any comment here?

Reply:
No, that was due to the automatic fix of vscode! I'll delete the comments right away